docs/upgrade-notes.asciidoc

```diff
@@ -0,0 +1,1714 @@
+[%collapsible]
+====
+*Details* +
+The bidirectional foreign key between agent policy and package policy has been removed. For more information, refer to {kibana-pull}138677[#138677].
+
+*Impact* +
+The agent policy saved object no longer includes the `package_policies` field.
+====
+
+[discrete]
+[[breaking-135669]]
+.[Fleet] xpack.agents.* are now uneditable in UI when defined in kibana.yml. (8.4)
+[%collapsible]
+====
+*Details* +
+When you configure `setxpack.fleet.agents.fleet_server.hosts` and `xpack.fleet.agents.elasticsearch.hosts` in kibana.yml, you are unable to update the fields on the Fleet UI.
+
+For more information, refer to {kibana-pull}135669[#135669].
+
+*Impact* +
+To configure `setxpack.fleet.agents.fleet_server.hosts` and `xpack.fleet.agents.elasticsearch.hosts` on the Fleet UI, avoid configuring the settings in kibana.yml.
+====
+
+[discrete]
+[[breaking-118854]]
+.[Fleet] Split package policy `upgrade` endpoint for Fleet. (8.0)
+[%collapsible]
+====
+*Details* +
+For package policy upgrades, the packagePolicy `upgrade` endpoint format supports a mutative upgrade operation (when `dryRun: false`) and a read-only dry run operation (when `dryRun: true`):
+
+[source,text]
+--
+ POST /package_policies/upgrade
+ {
+   packagePolicyIds: [...],
+   dryRun: false
+ }
+--
+
+For more information, refer to {kibana-pull}118854[#118854].
+
+*Impact* +
+The endpoint is now split into two separate endpoints:
+
+[source,text]
+--
+ POST /package_policies/upgrade
+ {
+   packagePolicyIds: [...]
+ }
+
+ POST /package_policies/upgrade/dry_run
+ {
+   packagePolicyIds: [...]
+ }
+--
+====
+
+// General settings
+
+[discrete]
+[[breaking-111535]]
+.[General settings] Removed `CONFIG_PATH` and `DATA_PATH` environment variables. (8.0)
+[%collapsible]
+====
+*Details* +
+The `CONFIG_PATH` and `DATA_PATH` environment variables have been removed. For more information, refer to {kibana-pull}111535[#111535].
+
+*Impact* +
+Replace the `CONFIG_PATH` environment variable with `KBN_PATH_CONF`, and replace `DATA_PATH` with the `path.data` setting.
+====
+
+[discrete]
+[[breaking-114379]]
+.[General settings] Removed support for csp.rules configuration. (8.0)
+[%collapsible]
+====
+*Details* +
+Support for the `csp.rules` configuration property has been removed. For more information, refer to {kibana-pull}114379[#114379].
+
+*Impact* +
+Configuring the default `csp.script_src`, `csp.workers_src`, and `csp.style_src` values is not required.
+====
+
+[discrete]
+[[breaking-113653]]
+.[General settings] Changed and removed deprecated core settings and deprecated settings from core plugins. (8.0)
+[%collapsible]
+====
+*Details* +
+The deprecation notice for `server.cors` has changed from `level:critical` to `level:warning`.
+
+The following settings have changed:
+
+* The `xpack.banners.placement` value has been renamed
+* The `newsfeed.defaultLanguage` newsfeed item retrieval method and default language has changed
```

Comment:
It's confusing having this line because the setting was removed as outlined below on line 440 and explained on line 450. Suggest removing this one

Reply:
```suggestion

```